javascript-modal.tar/javascript-modal/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <button id="open">Open Modal</button>
+  <div class="modalContainer" id="modalContainer">
+    <div class="modal">
+      <h1>Would you like to take a survey?</h1>
+      <button id="close">NO</button>
+    </div>
+  </div>
+</body>
```

Comment:
you're also missing the `script` tag to link your `main.js` file. Currently the page doesn't do anything since there is no javascript loaded